x-pack/plugins/security_solution/cypress/e2e/detection_alerts/expandable_flyout/alert_details_right_panel_overview_tab.cy.ts

```diff
@@ -51,74 +53,105 @@ describe.skip(
       openOverviewTab();
     });
 
-    it('should display correct title in header', () => {
-      cy.get(DOCUMENT_DETAILS_OVERVIEW_TAB_HEADER_TITLE)
-        .should('be.visible')
-        .and('have.text', rule.name);
-    });
-
-    it('should display risk score in header', () => {
-      cy.get(DOCUMENT_DETAILS_OVERVIEW_TAB_HEADER_RISK_SCORE).should('be.visible');
-      cy.get(DOCUMENT_DETAILS_OVERVIEW_TAB_HEADER_RISK_SCORE_VALUE)
-        .should('be.visible')
-        .and('have.text', rule.risk_score);
-    });
-
-    it('should display severity in header', () => {
-      cy.get(DOCUMENT_DETAILS_OVERVIEW_TAB_HEADER_SEVERITY).should('be.visible');
-      cy.get(DOCUMENT_DETAILS_OVERVIEW_TAB_HEADER_SEVERITY_VALUE)
-        .should('be.visible')
-        .and('have.text', upperFirst(rule.severity));
-    });
-
-    it('should display mitre attack', () => {
-      cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_MITRE_ATTACK_TITLE)
-        .should('be.visible')
-        // @ts-ignore
-        .and('contain.text', rule.threat[0].framework);
-
-      cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_MITRE_ATTACK_DETAILS)
-        .should('be.visible')
-        // @ts-ignore
-        .and('contain.text', rule.threat[0].technique[0].name)
-        // @ts-ignore
-        .and('contain.text', rule.threat[0].tactic.name);
-    });
+    describe('description section', () => {
+      it('should display description section header and content', () => {
+        cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_DESCRIPTION_SECTION_HEADER)
+          .should('be.visible')
+          .and('have.text', 'Description');
+        cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_DESCRIPTION_SECTION_CONTENT).should(
+          'be.visible'
+        );
+      });
 
-    it('should display highlighted fields', () => {
-      cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_HIGHLIGHTED_FIELDS).within(() => {
-        cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_HIGHLIGHTED_FIELDS_HEADER_EXPAND_ICON)
+      it('should display document description and expand button', () => {
+        cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_DESCRIPTION_TITLE)
           .should('be.visible')
-          .click();
-        cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_HIGHLIGHTED_FIELDS_HEADER_TITLE)
+          .and('have.text', 'Rule description');
+        cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_DESCRIPTION_DETAILS)
           .should('be.visible')
-          .and('have.text', 'Highlighted fields');
-        cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_HIGHLIGHTED_FIELDS_DETAILS).should(
-          'be.visible'
+          .and('have.text', rule.description);
+        cy.get(DOCUMENT_DETAILS_FLYOUT_OVERVIEW_TAB_DESCRIPTION_EXPAND_BUTTON)
```

Comment:
Will `Expand` always be visible?

Reply:
yes for now, like you noticed a bit below :)